application/electron/src/controllers/electron/controller.electron.menu.ts

```diff
@@ -126,15 +130,14 @@ export default class ControllerElectronMenu {
         }
         const exportActions: IExportAction[] = ServiceOutputExport.getActions(ServiceStreams.getActiveStreamId());
         if (exportActions.length > 0) {
-            template[0].submenu.push({ type: 'separator' });
+            template[1].submenu.push({ type: 'separator' });
             exportActions.forEach((action: IExportAction) => {
-                template[0].submenu.push({
+                template[1].submenu.push({
                     label: action.caption,
                     click: HandlerItemExportActionCall.bind(null, action.id),
                 });
             });
         }
-        template[0].submenu.push({ type: 'separator' });
         template[0].submenu.push({
             label: 'Plugins',
             click: HandlerItemPlugins,
```

Comment:
Move this inside the `if (darwin) {} else {}` block right below, and put it to main menu on darwin (right below Settings), and File menu on other platforms.

Reply:
Hi Florian, 
We've closed this pull-request without merge as soon as original issue #1080 isn't clean. This issue (#1080) should be described in more detail considering each platform. We should have more or less the same UI and user expectations for all platforms, but issue was creater more to macos direction.

Partly it's already implemented. For example, recent documents are already available via the dock context menu (mac) and taskbar (windows).